reactDashboard/src/components/SingleValue/SingleValueBar.css

```diff
@@ -24,22 +32,37 @@
 .single-value {
   box-shadow: 10px 10px 5px #aaaaaa;
   padding: 10px 0 0;
-  height: 100%;
+  height: 125px;
   background-color: #ffffff;
   color: #133a53;
   position: relative;
 }
 
+.darkmode .single-value {
+  background-color: rgb(56,56,56);
+  box-shadow: none;
+  color: #c1def1;
+}
+
 .region-single-value-bar,
 .single-value-bar {
-  padding: 10px;
+  padding: 0;
   margin: 0;
   margin-top: 0;
   display: flex;
+  justify-content: space-between;
   flex-direction: row;
   background-color: #f2f2f2;
   position: relative;
 }
+.darkmode .region-single-value-bar,
+.darkmode .single-value-bar {
+  background-color: #121212;
+}
+
+.darkmode .single-value-hr {
+  border: 1px solid #262626;
```

Comment:
Two things here - just override border-color, and have this block underneath the block it is overriding